app/controllers/v0/institutions_controller.rb

```diff
@@ -68,6 +68,12 @@ def search_results
                  .filter(:eight_keys, @query[:eight_keys_to_veteran_success]) # boolean
     end
 
+    # rubocop:disable Style/MutableConstant
```

Comment:
That's really strange. What kind of error? I don't think I've encountered this type of issue before with ruby constants and serialization.